procedure/pull-request-merge.rst

```diff
@@ -118,17 +121,23 @@ When you finish the checklist, decide whether to start
 #. Does the branch, and its merge, build and pass tests?
 
    CI should have run builds of both the branch, and a *trial merge*
-   of the pull request with master.  Success by CI is a strong
-   indication that `the merging procedure`_ will be quick and
-   successful.
+   [#trial-merge]_ of the pull request with master.  Recent success by
+   CI is a strong indication that `the merging procedure`_ will be
+   quick and successful.
 
    Look for build results in the pull request on GitHub.  Expand "Show
    all checks", and look for build success messages for both the
    branch and for the pull request (the trial merge).  If these
    results are missing, inform sysadmins that CI isn't functioning.
 
-   You can also look for a build results in the `Travis CI build
-   history for the repo`_ and in the `GitHub workflows for the repo`_.
+   You can also look for a build results in the logs of CI systems.
+   See `design.mps.ci.results <../design/tests.txt#ci-results>`__.
```

Comment:
m. This anchor seems to `#user-content-ci-results` in the beta of GitHub's code viewer.

Reply:
Confirmed by test by JPH. This is a problem introduced by the GitHub new code view beta.  I should tell them that they're breaking links.

Reply:
Raise: Provided feedback to GitHub https://github.com/orgs/community/discussions/48301#discussioncomment-5152498